resources/sdk/clisdkclient/templates/api.mustache

```diff
@@ -73,6 +73,11 @@ var {{operationId}}Cmd = &cobra.Command{
 		if len(queryParams) > 0 {
 			urlString = fmt.Sprintf("%v?", path)
 			for k, v := range queryParams {
+                if k == "varType" {
+                    k = "type"
+                    urlString += fmt.Sprintf("%v=%v&", url.QueryEscape(strings.TrimSpace(k)), url.QueryEscape(strings.TrimSpace(v)))
+                    continue
+                }
```

Comment:
Looks like the indentation is a bit off. You can fix this by opening this file in vs code and tabbing this block in and out again